docs/extensions/intermediate/controllers-routes.md

```diff
@@ -169,24 +170,16 @@ use Symfony\Component\HttpFoundation\Response;
  *
  * @author Kenny Koala <kenny@dropbear.com.au>
  */
-class DropBearController implements ControllerProviderInterface
+class DropBearController extends Base
 {
-    /** @var Application */
-    protected $app;
-
     /**
      * {@inheritdoc}
      */
-    public function connect(Application $app)
+    public function addRoutes(ControllerCollection $c)
```

Comment:
Interface [has it that way](https://github.com/bolt/bolt/blob/release/3.2/src/Controller/Base.php#L49), so it is consistent when viewed in the IDE hover.